torch/testing/_internal/common_utils.py

```diff
@@ -1257,17 +1257,26 @@ def assertEqualIgnoreType(self, *args, **kwargs) -> None:
     # TODO: default exact_device to True
     def assertEqual(self, x, y, msg: Optional[str] = None, *,
                     atol: Optional[float] = None, rtol: Optional[float] = None,
-                    equal_nan=True, exact_dtype=True, exact_device=False) -> None:
+                    equal_nan=True, exact_dtype=True, exact_device=False, exact_stride=False) -> None:
```

Comment:
Since strides were not checked before, we go with `False` be default.

Reply:
Which will also be default for `torch.testing.assert_close` after #60637 is landed.